src/cmd-generate-hashlist

```diff
@@ -0,0 +1,177 @@
+#!/usr/bin/env python3
+"""
+Creates a KeyLime Hashlist for an image.
+
+See: https://keylime.dev/
+"""
+
+import argparse
+import datetime
+import json
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+cosa_dir = os.path.dirname(os.path.abspath(__file__))
+sys.path.insert(0, cosa_dir)
+
+from cosalib import meta
+
+#: CREATE_HASH_IGN is the ignition we pass to kola to execute the hash
+#: creation script in the running system
+CREATE_HASH_IGN = """\
```

Comment:
I'd generate this from fcct to be (much) more readable.

Reply:
Good idea!